code/game/atoms_movable.dm

```diff
@@ -3,8 +3,6 @@
 	glide_size = 8
 	appearance_flags = TILE_BOUND|PIXEL_SCALE
 
-	///how many times a this movable had movement procs called on it since Moved() was last called
-	var/move_stacks = 0
```

Comment:
move_stacks should no longer be needed since connect_loc_behalf is no longer eating us alive. but we'll see, might revert

Reply:
didn't we need this to ensure signals were sent at the right time?